app/models/enforcement.rb

```diff
@@ -2,6 +2,9 @@
 
 class Enforcement < ApplicationRecord
   include Caseable
+
+  belongs_to :application_type
```

Comment:
I'm not sure about this. I realise RAB is representing it as a 'type' of planning application but that feels sort of like a workaround for them having to use the ODP submission schema, and I don't know if we gain anything by implementing the same workaround. Particularly when the 'type' is always going to be `breach`, which we can tell because it's stored in a different table; and any behaviour that would differ based on application type features won't be shared with enforcements anyway.

Is it going to be possible to handle this in the submission service without recording the type they send?